### ISSUE TITLE
PEP 541 - Package Index Name Retention for eventkit

### ISSUE THREAD

**Issue by gnzsnz** [ib-api-reloaded/eventkit#3]
issue created https://github.com/pypi/support/issues/4563

PEP 541 check list

## Reachability

* original author is not reachable, as unfortunately has passed away

## Abandoned projects criteria

All the criteria defined in pep 541 are meet

* [x] owner not reachable
* [x] no releases within the past twelve months; and
* [x] no activity from the owner on the project’s home page (or no home page listed).

## Continued maintenance of an abandoned project

* [x]     the project has been determined abandoned by the rules described above;
* [x]     the candidate is able to demonstrate their own failed attempts to contact the existing owner;
* [x]     the candidate is able to demonstrate improvements made on the candidate’s own fork of the project;
* [x]     the candidate is able to demonstrate why a fork under a different name is not an acceptable workaround; and
* [x]     the maintainers of the Package Index don’t have any additional reservations.

**Comment by mattsta** [comment]
oops, looks like we missed their reply window over there in https://github.com/pypi/support/issues/4563

We're left with one of these choices for long-term maintenance:

- reopen the pypi issue
- create a new pypi package like `aeventkit` (but then do we keep it `import eventkit` or change it everywhere to `import aeventkit`)
- reference eventkit by github URL as a dependency instead of a pypi package
- move eventkit directly inside the `ib_async` repo so it's not an external dependency anymore

I was looking at finally doing a new ib_async release soon with the new eventkit updates added, but we're stuck on where to pull eventkit from. Short term solution may just be a direct github URL dependency reference to get things moving then we could make a more permanent fix eventually.

**Comment by gnzsnz** [comment]
I don't think that a pypi issue will work. i would go with a new package `aeventkit` / `eventkit-reloaded` (or other suffix) and then `import eventkit`

let me know if you need any help, i'm willing to work on this and ib_async.

**Comment by mattsta** [comment]
Yeah, new package name is probably the easiest. I'll look into that next and figure out better distributed ways of updating things.

For now just going with the temporary hack of directly linking to a github tag as a dependency so there's no more waiting for a release update.


Thanks for your continual attention and feedback on everything! Hopefully we can get to a more incremental update/test/release cycle soon between projects and packages.

Since everything is on a new 2.0 version cycle now, we can re-evaluate any open issues or PRs everywhere and see what still matters for inclusion/refactoring or what should be dropped.

**Comment by mattsta** [comment]
oops, my temporary fix as a direct github dependency from ib_async to here didn't work, so looks like a new aeventkit package is on the way to get a new release out:

```haskell
HTTP Error 400: Can't have direct dependency:
eventkit@ git+https://github.com/ib-api-reloaded/eventkit@v2.0.0.
See https://packaging.python.org/specifications/core-metadata for more information.
```
